util/encoding/decimal.go

```diff
@@ -356,24 +394,23 @@ var digitsLookupTable [tableSize + 1]tableVal
 
 type tableVal struct {
 	digits int
-	border *big.Int
+	border big.Int
 }
 
 func init() {
-	digitBi := new(big.Int)
-	var bigIntArr [tableSize]big.Int
+	curVal := big.NewInt(1)
```

Comment:
`var curExp big.Int` wouldn't actually do anything because the second I address it, it will escape to the heap anyway. Plus, everything in this `init` function is done on startup, so performance basically doesnt matter.